{
  "Mayavi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2300546",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291477"
  ],
  "NLopt": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2254979"
  ],
  "blender": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291492"
  ],
  "csound": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271396"
  ],
  "fedmsg": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2300669",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291518"
  ],
  "python-datadog": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290567"
  ],
  "python-gbulb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291712"
  ],
  "python-jnius": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272966"
  ],
  "python-multiprocess": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246347"
  ],
  "python-pdir2": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275052"
  ],
  "python-ppft": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291847"
  ],
  "python-pytn3270": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246275"
  ],
  "python-pyunicorn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301203",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291888"
  ],
  "python-qcustomplot-pyqt": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301204",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291892"
  ],
  "python-ufo2ft": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301217",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291961"
  ],
  "python-webpy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2256932"
  ],
  "python-x2go": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301218"
  ],
  "ubertooth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259655"
  ],
  "xonsh": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2283616"
  ]
}